.evergreen/run-mongodb-oidc-test.sh

```diff
@@ -30,7 +30,7 @@ export TEST_AUTH_OIDC=1
 export COVERAGE=1
 export AUTH="auth"
 
-cargo test test::spec::oidc -- --nocapture --test-threads=1
+cargo nextest run test::spec::oidc
```

Comment:
This should use the `ci` profile, and it'll also be good to have junit output so evergreen shows us a test list rather than a text blob.
```
cargo nextest run test::spec::oidc --profile ci
cp target/nextest/ci/junit.xml results.xml
```
should do it.

Reply:
sgtm, I'll try. Thanks!